webpack/shared/resolve.js

```diff
@@ -5,17 +5,10 @@ module.exports = {
   alias: {
     actions: path.resolve(__dirname, '../../src/actions'),
     api: path.resolve(__dirname, '../../src/api'),
-    assets: path.resolve(__dirname, '../../src/assets'),
     components: path.resolve(__dirname, '../../src/components'),
     constants: path.resolve(__dirname, '../../src/constants'),
     pages: path.resolve(__dirname, '../../src/pages'),
-    services: path.resolve(__dirname, '../../src/services'),
-    hooks: path.resolve(__dirname, '../../src/hooks'),
-    locales: path.resolve(__dirname, '../../src/locales'),
-    reducers: path.resolve(__dirname, '../../src/reducers'),
     selectors: path.resolve(__dirname, '../../src/selectors'),
-    store: path.resolve(__dirname, '../../src/store'),
-    styles: path.resolve(__dirname, '../../src/styles'),
```

Comment:
why are these being deleted? how are you going to organize your files moving forward?

Reply:
assets/services/hooks/styles folders don't exist anymore. store/reducers/locales was being used only in one place of the app. And well, if I need something else in the future, I will add them, but I don't want to have code that doesn't make anything special

Reply:
Why won't you have `services`? or `hooks`?  styles, i get it, you are using styled so thats 👌. 
But the rest seems to me like you are breaking the whole module architecture, that's why the question, if those are not there, how are you going to structure your modules? where are you going to put your api integrations, custom hooks, reducers, etc? 

This project will be eventually open-sourced an mantained by the whole react team, so we need to keep things consistent with company standards at least by some margin, otherwise any team member that has to work on this will have to re-learn a ton of stuff.

Reply:
If I had to add new hooks, I will add it this way for sure, I'm not going to go against that, but I wanted to remove it because I don't use custom hooks atm.
I'm new at Rootstrap, so I have a really different background than the people who work in here. I don't really want to cause discomfort, this thing I'm doing is because I'm trying to keep everything simple. I have used a lot of different stuff in the React that has changed like a lot, is something that is going on with improvement all the time. Please, try to understand that I don't mean harm. But I also don't know how you really work guys. I will try my best to follow your guidelines, and sometimes I can mess up. But if I see I can make a change that feels worth it for me, I will try to do it. And if it is not okay, you can always go back.

Reply:
Sounds good 🙂 the idea is to keep the discussion ongoing, there's always room for improvement (both on the base projects and on any other), so don't stop proposing changes 💪 .

Reply:
Make sense, I'm really gratefully that you take your time to have a look at this and worry about changing stuff